actions/utils.js

```diff
@@ -214,7 +214,13 @@ async function getConfig(context) {
   if (!context.config) {
     logger.debug(`Fetching config ${configName}`);
     const configData = await requestSpreadsheet(configName, null, context);
-    context.config = configData.data.reduce((acc, { key, value }) => ({ ...acc, [key]: value }), {});
+    if(configData.data) {
+      context.config = configData.data.reduce((acc, { key, value }) => ({ ...acc, [key]: value }), {});
+      context.config.__hasLegacyFormat = true;
```

Comment:
should we make config service default, and if no config is found there it falls back to spreadsheet configs.json?